app/controllers/verify_controller.rb

```diff
@@ -0,0 +1,7 @@
+class VerifyController < ApplicationController
+  include RenderConditionConcern
+
+  render_if -> { IdentityConfig.store.idv_api_enabled }, only: [:show]
```

Comment:
still love this, another idea I have for the name:

```suggestion
  check_or_render_not_found -> { IdentityConfig.store.idv_api_enabled }, only: [:show]
```

Reply:
> still love this, another idea I have for the name:

Would the idea be to emphasize the actual behavior of `render_not_found`?

I could also imagine:

```suggestion
  render_not_found_unless -> { IdentityConfig.store.idv_api_enabled }, only: [:show]
```

Or:

```suggestion
  render_not_found_if -> { !IdentityConfig.store.idv_api_enabled }, only: [:show]
```

Reply:
I also sort of wonder if it would be so bad to just have the complete logic here, if the concern is concealing too much.

```suggestion
  before_action { render_not_found if !IdentityConfig.store.idv_api_enabled }, only: [:show]
```

Reply:
Went with proposed rename to keep this moving along. Can always come back to it later.

Reply:
Makes sense! We have a bunch of similar code we could refactor all at once too -- did we file a followup ticket?

Reply:
> We have a bunch of similar code we could refactor all at once too -- did we file a followup ticket?

We did now! See LG-6157.